webpackJsonp([0],{H7s1:function(e,t){},Ii0A:function(e,t,a){e.exports=a.p+"static/img/matrix.a60a3cc.png"},NHnr:function(e,t,a){"use strict";Object.defineProperty(t,"__esModule",{value:!0});var i=a("7+uW"),s={render:function(){var e=this.$createElement,t=this._self._c||e;return t("div",{attrs:{id:"app"}},[t("router-view")],1)},staticRenderFns:[]};var n=a("VU/8")({name:"App"},s,!1,function(e){a("OcnN")},null,null).exports,r=a("/ocq"),l={name:"portal",mounted:function(){window.addEventListener("scroll",this.handleScroll)},data:function(){return{navActive:"Home"}},methods:{changeActive:function(e){switch(this.navActive=e,e){case"Career":this.$el.querySelector("#Career").scrollIntoView({block:"start"});break;case"Cell_Atlas":this.$el.querySelector("#Cell_Atlas").scrollIntoView({block:"start"})}},handleScroll:function(){var e=window.pageYOffset||document.documentElement.scrollTop||document.body.scrollTop;this.navActive=e<700?"Home":e>=700&&e<1600?"Career":"Cell_Atlas"},toReference:function(){this.$el.querySelector("#Reference").scrollIntoView({block:"start"})}}},o={render:function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{attrs:{id:"portal"}},[a("nav",{staticClass:"navbar navbar-expand-lg navbar-light compensate-for-scrollbar",attrs:{id:"navbar-main"}},[a("div",{staticClass:"container"},[e._m(0),e._v(" "),e._m(1),e._v(" "),e._m(2),e._v(" "),a("div",{staticClass:"navbar-collapse main-menu-item collapse justify-content-start",attrs:{id:"navbar-content"}},[a("ul",{staticClass:"navbar-nav d-md-inline-flex"},[a("li",{staticClass:"nav-item"},[a("a",{staticClass:"nav-link",class:{active:"Home"==e.navActive},attrs:{href:"/Buhan_Feng","data-target":"#Home"},on:{click:function(t){return e.changeActive("Home")}}},[a("span",[e._v("Home")])])]),e._v(" "),a("li",{staticClass:"nav-item"},[a("a",{staticClass:"nav-link",class:{active:"Career"==e.navActive},staticStyle:{cursor:"pointer"},on:{click:function(t){return e.changeActive("Career")}}},[a("span",[e._v("Career")])])]),e._v(" "),a("li",{staticClass:"nav-item"},[a("a",{staticClass:"nav-link ",class:{active:"Cell_Atlas"==e.navActive},staticStyle:{cursor:"pointer"},on:{click:function(t){return e.changeActive("Cell_Atlas")}}},[a("span",[e._v("Cell Atlas")])])]),e._v(" "),e._m(3),e._v(" "),e._m(4)])])])]),e._v(" "),a("section",{staticClass:"home-section wg-about   ",attrs:{id:"about"}},[a("div",{staticClass:"container"},[a("div",{staticClass:"row"},[e._m(5),e._v(" "),a("div",{staticClass:"col-12 col-lg-8"},[e._m(6),e._v(" "),a("p",[e._v("I am looking for Research Assistant position toward my interested subjects. RA service is voluntary, offering roughly 15 hours working time per week, produce can be delivered weekly or semi-monthly. Besides, I will finish MPhil course on June 2023, so I am also seeking 2023 fall Ph.D. position. "),a("a",{staticClass:"details",on:{click:function(t){return e.changeActive("Career")}}},[e._v("More details.")])]),e._v(" "),a("p",[e._v("I am also watching for mathematical cooperation/guidance on a personal project (A Comprehensive Developmental Cell Atlas for Zebrafish). In general, original data was collected and processed to generate a 25,000*990,000 (features*samples) matrix. We have several exciting problems to answer through analyzing this matrix. Data is free to explore.  "),a("a",{staticClass:"details",on:{click:function(t){return e.changeActive("Cell_Atlas")}}},[e._v("More details.")])]),e._v(" "),e._m(7)])])])]),e._v(" "),e._m(8),e._v(" "),a("section",{staticClass:"home-section wg-about",attrs:{id:"Cell_Atlas"}},[a("div",{staticClass:"container"},[a("div",{staticClass:"row"},[a("div",{staticClass:"col-12"},[a("h2",{staticStyle:{"font-weight":"bold","margin-bottom":"50px"}},[e._v("A Comprehensive Developmental Cell Atlas for Zebrafish")]),e._v(" "),a("div",{staticClass:"row"},[e._m(9),e._v(" "),e._m(10),e._v(" "),a("div",{staticStyle:{width:"100%",height:"15px"}}),e._v(" "),e._m(11),e._v(" "),a("div",{staticClass:"col-9"},[a("p",{},[a("span",{staticClass:"lead-emphasis"},[e._v("T")]),e._v("o promote the understanding of human cellular heterogeneity, there is a proposal of a convention, Human Cell Atlas (HCA, "),a("a",{attrs:{href:"https://www.humancellatlas.org",target:"_blank"}},[e._v("www.humancellatlas.org")]),e._v(")"),a("a",{staticClass:"citation-index",on:{click:function(t){return e.toReference()}}},[e._v("(1-2)")]),e._v(". Experience from different labs around the world which comply to the convention can be utilized to build human cell atlas. Human Cell Atlas is another mega project after "),a("a",{attrs:{href:"https://www.genome.gov/human-genome-project",target:"_blank"}},[e._v("The Human Genome Project")]),e._v(".")])]),e._v(" "),a("div",{staticStyle:{width:"100%",height:"15px"}}),e._v(" "),e._m(12),e._v(" "),e._m(13),e._v(" "),a("div",{staticStyle:{width:"100%",height:"15px"}}),e._v(" "),e._m(14),e._v(" "),a("div",{staticClass:"col-9"},[e._m(15),e._v(" "),a("p",{},[e._v("The other reason for choosing zebrafish to build cell atlas is that zebrafish is more scalable compare to human. On contrast to human cell atlas which need labor from labs around the world, zebrafish is reachable one. One of aspect, for instance, is this smaller shape at anatomy which substantially need much lower cell coverage.")]),e._v(" "),a("p",{},[e._v("To date, dozens of studies relate to zebrafish which apply sc-RNA-seq was published. Among these studies, some independently investigate a specific stage of embryonic development "),a("a",{staticClass:"citation-index",on:{click:function(t){return e.toReference()}}},[e._v("(3-5)")]),e._v(". Other studies obtain several samples from a developmental window (typically 0-24 hours) with designated sampling interval "),a("a",{staticClass:"citation-index",on:{click:function(t){return e.toReference()}}},[e._v("(6, 7)")]),e._v(". Because sc-RNA-seq is costly, these works either failed to screen long developmental window or designed a long sample interval which leads to low cell coverage.")]),e._v(" "),a("p",{staticClass:"content-emphasis"},[e._v("Overall, zebrafish is a valuable model for investigating cellular heterogeneity. Although different stage of zebrafish embryo has been study, a more comprehensive view of zebrafish developmental cell atlas is lacking.")])]),e._v(" "),a("div",{staticStyle:{width:"100%",height:"15px"}}),e._v(" "),e._m(16),e._v(" "),e._m(17),e._v(" "),a("div",{staticStyle:{width:"100%",height:"15px"}}),e._v(" "),e._m(18),e._v(" "),a("div",{staticClass:"col-9"},[e._m(19),e._v(" "),a("ul",[e._m(20),e._v(" "),e._m(21),e._v(" "),e._m(22),e._v(" "),a("li",[e._m(23),e._v(" "),e._m(24),e._v(" "),a("p",[e._v("Overall, extra data is needed to promote the performance of sc-RNA-seq data. One of the prominent data type is spatial RNA sequencing data, starting from 2021, some technologies can obtain subcellular resolution "),a("a",{staticClass:"citation-index",on:{click:function(t){return e.toReference()}}},[e._v("(8, 9)")]),e._v(", making this kinds of technology has more exciting usage. For more discussion, please kindly refer to "),a("a",{staticStyle:{cursor:"not-allowed"},attrs:{href:"javascript:;"}},[e._v("SpatioBorder")]),e._v(". ")])])])]),e._v(" "),a("div",{staticStyle:{width:"100%",height:"15px"}}),e._v(" "),e._m(25),e._v(" "),e._m(26),e._v(" "),a("div",{staticStyle:{width:"100%",height:"15px"}}),e._v(" "),e._m(27),e._v(" "),e._m(28)])])])])])])},staticRenderFns:[function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"d-none d-lg-inline-flex"},[t("a",{staticClass:"navbar-brand",attrs:{href:"/"}},[this._v("Buhan Feng")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("button",{staticClass:"navbar-toggler",attrs:{type:"button","data-toggle":"collapse","data-target":"#navbar-content","aria-controls":"navbar","aria-expanded":"false","aria-label":"Toggle navigation"}},[t("span",[t("i",{staticClass:"fas fa-bars"})])])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"navbar-brand-mobile-wrapper d-inline-flex d-lg-none"},[t("a",{staticClass:"navbar-brand",attrs:{href:"/"}},[this._v("Buhan Feng")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("li",{staticClass:"nav-item"},[t("a",{staticClass:"nav-link ",staticStyle:{cursor:"not-allowed"},attrs:{href:"javascript:;","data-target":"#SpatioBorder"}},[t("span",[this._v("SpatioBorder")])])])},function(){var e=this.$createElement,t=this._self._c||e;return t("li",{staticClass:"nav-item"},[t("a",{staticClass:"nav-link",attrs:{href:"../../static/FENG_Buhan_CV.pdf",target:"_blank"}},[t("span",[this._v("CV")])])])},function(){var e=this,t=e.$createElement,i=e._self._c||t;return i("div",{staticClass:"col-12 col-lg-4"},[i("div",{attrs:{id:"profile"}},[i("img",{staticClass:"portrait",attrs:{src:a("bnyv"),alt:""}}),e._v(" "),i("div",{staticClass:"portrait-title"},[i("h2",[e._v("Buhan Feng")]),e._v(" "),i("h3",[e._v("Postgraduate")]),e._v(" "),i("h3",[i("a",{attrs:{href:"https://lifesciences.sysu.edu.cn/",target:"_blank",rel:"noopener"}},[i("span",[e._v("School of Life Science, SYSU")])])])]),e._v(" "),i("ul",{staticClass:"network-icon",attrs:{"aria-hidden":"true"}},[i("li",[i("a",{attrs:{href:"mailto:buhan_feng@foxmail.com",title:"Left mouse button to send mail or right mouse button to copy mail address."}},[i("i",{staticClass:"fa-solid fa-envelope  big-icon"})])]),e._v(" "),i("li",[i("a",{attrs:{href:"https://github.com/VonImt",target:"_blank",rel:"noopener"}},[i("i",{staticClass:"fab fa-github big-icon"})])])])])])},function(){var e=this.$createElement,t=this._self._c||e;return t("p",[this._v("I am a Postgraduate student at Sun Yat-sen University, major in Bioinformatics. During 2016 to 2020, I served as software developer on Teligen Co.Ltd (a state-owned enterprise). I obtained my B.E. in Electronic Engineering, 2016, from Southern China Agriculture University. "),t("a",{attrs:{href:"../../static/FENG_Buhan_CV.pdf",target:"_blank"}},[this._v("More details.")])])},function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"row"},[a("div",{staticClass:"col-md-7"},[a("h3",[e._v("Education")]),e._v(" "),a("ul",{staticClass:"ul-edu fa-ul"},[a("li",[a("i",{staticClass:"fa-li fas fa-graduation-cap"}),e._v(" "),a("div",{staticClass:"description"},[a("p",{staticClass:"course"},[e._v("M.S. in Bioinformatics, 2023")]),e._v(" "),a("p",{staticClass:"institution"},[a("a",{attrs:{href:"https://www.sysu.edu.cn/",target:"_blank"}},[e._v("Sun Yat-sen University")])])])]),e._v(" "),a("li",[a("i",{staticClass:"fa-li fas fa-graduation-cap"}),e._v(" "),a("div",{staticClass:"description"},[a("p",{staticClass:"course"},[e._v("B.E. in Electronic Engineering, 2016")]),e._v(" "),a("p",{staticClass:"institution"},[a("a",{attrs:{href:"https://www.scau.edu.cn/",target:"_blank"}},[e._v("South China Agricultural University")])])])])])])])},function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("section",{staticClass:"home-section wg-about",attrs:{id:"Career"}},[a("div",{staticClass:"container"},[a("div",{staticClass:"row"},[a("h2",{staticStyle:{"font-weight":"bold","margin-bottom":"50px"}},[e._v("Career")]),e._v(" "),a("p",[e._v("I am now working on a biological "),a("a",{staticClass:"query",attrs:{href:"https://en.wikipedia.org/wiki/Wet_lab",target:"_blank"}},[e._v("wet lab")]),e._v(", my current work is unsaturated, so I looking for a chance to take part in extra project to deepen my understanding and make best use of my off-duty time.")]),e._v(" "),a("p",[e._v("Here are some demonstrations of task that I am adept at: ")]),e._v(" "),a("ul",[a("li",[a("p",[e._v("Software development: ")]),e._v(" "),a("p",[e._v("I have program experience on many fields, my major is web application development (both front and back end), which I can lead a middle scale project.")]),e._v(" "),a("p",[e._v("For other projects which may use C/C++, python, R for some reasons, I can develop individual module complying with convention.")])]),e._v(" "),a("li",[a("p",[e._v("Biological data analysis:")]),e._v(" "),a("p",[e._v("I have experience on processing different kinds of sequencing data, such as single-cell RNA, bulk RNA, 16S, Hi-C etc. I would also like to try on some new data type if having the chance.")])])]),e._v(" "),a("p",{staticStyle:{display:"block"}},[e._v("My interested subjects include: "),a("span",{staticClass:"content-emphasis"},[e._v("Application of Machine Learning/Scientific Software Development/Data Analysis.")]),e._v(" For the past years, I have been received many cross-disciplinary training, except for above-mentioned interest, I also keep open mind to other relative fields, it would be appreciated if I obtain working opportunity which I do not familiar with.")]),e._v(" "),a("p",[e._v("I hope labs which may offer me RA position also has PhD quota and considering to recruit me as PhD student on 2023 fall.")])])])])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"col-3"},[t("h4",{staticClass:"subtitle"},[this._v("What Is Cell Atlas?")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"col-9"},[t("p",{},[t("span",{staticClass:"lead-emphasis"},[this._v("C")]),this._v("ell atlas is a way to illustrate "),t("a",{staticClass:"query",attrs:{href:"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2918286",target:"_blank"}},[this._v("cellular heterogeneity")]),this._v(" within an organism, cellular heterogeneity is a biological term which reflects distinct cell characters in different organ, or different development stage of an organ.")]),this._v(" "),t("p",{},[this._v("Understanding of cellular heterogeneity has long been a core challenge for several biological subjects including Developmental Biology, Regenerative Medicine, Immunology, etc. And therefore has great influence on Precise Medicine, organ transplant and other biological engineering.")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"col-3"},[t("h4",{staticClass:"subtitle"},[this._v("Human Cell Atlas Consortium")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"col-3"},[t("h4",{staticClass:"subtitle"},[this._v("Single Cell RNA Sequencing")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"col-9"},[t("p",{},[t("a",{staticClass:"query",attrs:{href:"https://en.wikipedia.org/wiki/Single-cell_transcriptomics",target:"_blank"}},[t("span",{staticClass:"lead-emphasis"},[this._v("S")]),this._v("ingle cell RNA sequencing")]),this._v(" (hereafter referred to as sc-RNA-seq) technology boosted since 2016, it is capable to dissect "),t("a",{staticClass:"query",attrs:{href:"https://en.wikipedia.org/wiki/Transcriptome",target:"_blank"}},[this._v("transcriptome")]),this._v(" from distinct cell (single cell resolution), and process hundreds of cells parallelly. Cellular heterogeneity can be largely reflected by cell's transcriptome, so sc-RNA-seq is a prominent technology to investigate cellular heterogeneity. sc-RNA-seq data can be used to constructing cell atlas.")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"col-3"},[t("h4",{staticClass:"subtitle"},[this._v("Why Zebrafish?")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("p",{},[t("a",{staticClass:"query",attrs:{href:"https://en.wikipedia.org/wiki/Zebrafish",target:"_blank"}},[t("span",{staticClass:"lead-emphasis"},[this._v("Z")]),this._v("ebrafish")]),this._v(" is one of the Model Organism at biology, Human and Zebrafish has more than 80% similarity on genome. consequently, depicting molecular mechanism in zebrafish potentially enlightening research on human and medicine.")])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"col-3"},[t("h4",{staticClass:"subtitle"},[this._v("Data Availability")])])},function(){var e=this,t=e.$createElement,i=e._self._c||t;return i("div",{staticClass:"col-9"},[i("p",{},[i("span",{staticClass:"lead-emphasis"},[e._v("T")]),e._v("otal 4 TB data was gathered, most of which is sc-RNA-seq data, after quality control -> mapping reads to reference genome -> counting, raw data was transform into a 25,000*990,000 (row*column) matrix, which row and column can refer to as features and samples respectively. Here is an illustration of the matrix:")]),e._v(" "),i("div",{staticClass:"fig-wrap"},[i("div",{staticClass:"pic-wrap",staticStyle:{width:"600px"}},[i("img",{attrs:{src:a("Ii0A")}})]),e._v(" "),i("div",{staticClass:"fig-comment"},[e._v("In the context of sc-RNA-seq data, features equivalent to genes and samples equivalent to cells.")])]),e._v(" "),i("p",{},[e._v("Matrix data was persisted on my "),i("a",{attrs:{href:"https://github.com/VonImt",target:"_blank"}},[e._v("github repository")]),e._v(". Raw data is too big to persist on serve (4 TB after compressing), so it is not provided online. Several state-of-the-art pipelines for mapping and counting procedure was run and evaluate consistence between their results. For who was interested in raw data or processing protocol, please contact me.")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"col-3"},[t("h4",{staticClass:"subtitle"},[this._v("Research Goals")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("p",{},[t("span",{staticClass:"lead-emphasis"},[this._v("B")]),this._v("elow is some problems we can investigate from the matrix:")])},function(){var e=this.$createElement,t=this._self._c||e;return t("li",[t("p",[this._v("Evaluating Integrative Ability:")]),this._v(" "),t("p",[this._v("Because data are coming from different experience, we need a quantitative model to assess the power of batch effects and integrative ability between the data sets. Then we removing batch effects for further analysis.")])])},function(){var e=this,t=e.$createElement,i=e._self._c||t;return i("li",[i("p",[e._v("Building Differential Trajectory:")]),e._v(" "),i("p",[e._v("The development of vertebrate embryo is a continuously differential procedure, a trajectory can be built to indicate this procedure, below is a demonstration of trajectory.")]),e._v(" "),i("div",{staticClass:"fig-wrap"},[i("div",{staticClass:"pic-wrap",staticStyle:{width:"500px"}},[i("img",{attrs:{src:a("dNub")}})]),e._v(" "),i("div",{staticClass:"fig-comment"},[e._v("Shape of a trajectory is similar to a growing tree, which root and branch can be recognized as stem cells and differential cells, each forks indicating a differential issue. As developmental time grows up, more distinctly differential cells can be observed (more branches).")])]),e._v(" "),i("p",[e._v("Base on the trajectory, we can also identify which features (genes) are dominating at each specify branch.")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("li",[t("p",[this._v("database and visualization:")]),this._v(" "),t("p",[this._v("Outcomes of this research will organized as a database. We integrate visualization engine and other useful API into a tool kit, this tool kit can either access this database or processing data that come from other projects.")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("p",[t("a",{staticClass:"query",attrs:{href:"https://en.wikipedia.org/wiki/Omics",target:"_blank"}},[this._v("Multi-omic")]),this._v(" Analysis:")])},function(){var e=this.$createElement,t=this._self._c||e;return t("p",[this._v("Sc-RNA-seq is a major method to dissect cellular heterogeneity, but there are three reasons that limited its capability to totally reflect cellular heterogeneity. First, RNA molecular amount within a cell is dynamic (also known as "),t("a",{staticClass:"query",attrs:{href:"https://pubmed.ncbi.nlm.nih.gov/30089906/",target:"_blank"}},[this._v("RNA velocity")]),this._v("), but sc-RNA-seq technology only screen RNA profile at one specific time point. Second, a specific gene expresses high amount of RNA don not equivalent to express high amount of protein, because RNA typically undergoes "),t("a",{staticClass:"query",attrs:{href:"https://en.wikipedia.org/wiki/Post-translational_modification",target:"_blank"}},[this._v("post-translational modification")]),this._v(" and degrade procedures. Third, although sc-RNA-seq data can get RNA profile at single cell resolution, but it fails to reflect cell's original spatial location.")])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"col-3"},[t("h4",{staticClass:"subtitle"},[this._v("Significance")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"col-9"},[t("p",[t("span",{staticClass:"lead-emphasis"},[this._v("T")]),this._v("aken together, our research offers unprecedently comprehensive view on development of zebrafish. Besides, we provide valuable resource for the single-cell community to develop and evaluate new strategies for an informative and interpretable cell atlas.")])])},function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"col-3",attrs:{id:"Reference"}},[t("h4",{staticClass:"subtitle"},[this._v("Reference")])])},function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"col-9 citation-content"},[a("p",[e._v("1. "),a("span",{staticClass:"content-emphasis"},[e._v("Human Cell Atlas.")]),e._v(" Home, https://www.humancellatlas.org/ (2021).")]),e._v(" "),a("p",[e._v("2. Haniffa, M., Taylor, D., Linnarsson, S. et al. "),a("span",{staticClass:"content-emphasis"},[e._v("A roadmap for the Human Developmental Cell Atlas. Nature")]),e._v(" 597, 196–205 (2021). https://doi.org/10.1038/s41586-021-03620-1.")]),e._v(" "),a("p",[e._v("3. Farnsworth DR, Saunders LM, Miller AC. "),a("span",{staticClass:"content-emphasis"},[e._v("A single-cell transcriptome atlas for zebrafish development. Dev Biol.")]),e._v(" 2020 Mar 15;459(2):100-108. doi: 10.1016/j.ydbio.2019.11.008. Epub 2019 Nov 27. PMID: 31782996; PMCID: PMC7080588.")]),e._v(" "),a("p",[e._v("4. Metikala S, Casie Chetty S, Sumanas S. "),a("span",{staticClass:"content-emphasis"},[e._v("Single-cell transcriptome analysis of the zebrafish embryonic trunk. PLoS One.")]),e._v(" 2021 Jul 7;16(7):e0254024. doi: 10.1371/journal.pone.0254024. PMID: 34234366; PMCID: PMC8263256.")]),e._v(" "),a("p",[e._v("5. Xia J, Kang Z, Xue Y, Ding Y, Gao S, Zhang Y, Lv P, Wang X, Ma D, Wang L, Han JJ, Liu F. "),a("span",{staticClass:"content-emphasis"},[e._v("A single-cell resolution developmental atlas of hematopoietic stem and progenitor cell expansion in zebrafish. Proc Natl Acad Sci U S A.")]),e._v(" 2021 Apr 6;118(14):e2015748118. doi: 10.1073/pnas.2015748118. PMID: 33785593; PMCID: PMC8040670.")]),e._v(" "),a("p",[e._v("6. Farrell JA, Wang Y, Riesenfeld SJ, Shekhar K, Regev A, Schier AF. "),a("span",{staticClass:"content-emphasis"},[e._v("Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis. Science.")]),e._v(" 2018 Jun 1;360(6392):eaar3131. doi: 10.1126/science.aar3131. Epub 2018 Apr 26. PMID: 29700225; PMCID: PMC6247916.")]),e._v(" "),a("p",[e._v("7. Wagner DE, Weinreb C, Collins ZM, Briggs JA, Megason SG, Klein AM. "),a("span",{staticClass:"content-emphasis"},[e._v("Single-cell mapping of gene expression landscapes and lineage in the zebrafish embryo. Science.")]),e._v(" 2018 Jun 1;360(6392):981-987. doi: 10.1126/science.aar4362. Epub 2018 Apr 26. PMID: 29700229; PMCID: PMC6083445.")]),e._v(" "),a("p",[e._v("8. Cho, C.S., Xi, J., Si, Y., Park, S.R., Hsu, J.E., Kim, M., Jun, G., Kang, H.M., and Lee, J.H. (2021a). "),a("span",{staticClass:"content-emphasis"},[e._v("Microscopic examination of spatial transcriptome using Seq-Scope. Cell.")]),e._v(" doi:10.1016/j.cell.2021.05.010")]),e._v(" "),a("p",[e._v("9. Chen A, Liao S, Cheng M, Ma K, Wu L, Lai Y, Qiu X, Yang J, Xu J, Hao S, Wang X, Lu H, Chen X, Liu X, Huang X, Li Z, Hong Y, Jiang Y, Peng J, Liu S, Shen M, Liu C, Li Q, Yuan Y, Wei X, Zheng H, Feng W, Wang Z, Liu Y, Wang Z, Yang Y, Xiang H, Han L, Qin B, Guo P, Lai G, Muñoz-Cánoves P, Maxwell PH, Thiery JP, Wu QF, Zhao F, Chen B, Li M, Dai X, Wang S, Kuang H, Hui J, Wang L, Fei JF, Wang O, Wei X, Lu H, Wang B, Liu S, Gu Y, Ni M, Zhang W, Mu F, Yin Y, Yang H, Lisby M, Cornall RJ, Mulder J, Uhlén M, Esteban MA, Li Y, Liu L, Xu X, Wang J. "),a("span",{staticClass:"content-emphasis"},[e._v("Spatiotemporal transcriptomic atlas of mouse organogenesis using DNA nanoball-patterned arrays. Cell.")]),e._v(" 2022 May 12;185(10):1777-1792.e21. doi: 10.1016/j.cell.2022.04.003. Epub 2022 May 4. PMID: 35512705.")])])}]};var c=a("VU/8")(l,o,!1,function(e){a("H7s1")},"data-v-3b4c25d0",null).exports;i.a.use(r.a);var h=new r.a({routes:[{path:"/",name:"portal",component:c}]});i.a.config.productionTip=!1,new i.a({el:"#app",router:h,components:{App:n},template:"<App/>"})},OcnN:function(e,t){},bnyv:function(e,t,a){e.exports=a.p+"static/img/Buhan_Feng.de9fd8d.jpg"},dNub:function(e,t,a){e.exports=a.p+"static/img/tra.f474adb.png"}},["NHnr"]);
//# sourceMappingURL=app.14b877f071b2401338b9.js.map